package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "terra-ui_com",
+  "name": "terra-ui.com",
```

Comment:
If anyone is curious like I was, `.` is allowed in package names. https://docs.npmjs.com/files/package.json

Reply:
the '_' broke the generate-config pattern which caused the change initially